src/tests/test_console.py

```diff
@@ -229,7 +229,7 @@ def test_nonpersistent_redirection(safe_sys_redirections):
     my_stdout = ""
     my_stderr = ""
 
-    def stdin_callback():
+    def stdin_callback(string):
```

Comment:
This isn't right. `stdin_callback` should take no inputs and return a string. I guess the problem is that line 228 in `src/py/_pyodide/console.py` is wrong. It currently reads:
```py
stdin_callback: Optional[Callable[[str], None]] = None,
```
and should be changed to:
```py
stdin_callback: Optional[Callable[[], str]] = None
```

Reply:
I definitely thought mypy was running on these files.

Reply:
Ahh, yes, that's the correct fix. MyPy only runs on things that have types (until this PR, which runs it on untyped functions and classes)

Reply:
Right hmm. Is there a flag to require types for everything? I suppose we could enable that in a subsequent PR.

Reply:
That's

```
check_untyped_defs = true
```

And it's the main feature of this PR. :)

Reply:
> And it's the main feature of this PR.

Oh... right. Thanks for your patience @henryiii :laughing: